test/e2e/tests/settings/address-book.spec.js

```diff
@@ -70,6 +70,61 @@ describe('Address Book', function () {
       },
     );
   });
+
+  it('Adds a new contact to the address book', async function () {
+    await withFixtures(
+      {
+        fixtures: new FixtureBuilder().build(),
+        ganacheOptions: defaultGanacheOptions,
+        title: this.test.fullTitle(),
+      },
+      async ({ driver }) => {
+        await unlockWallet(driver);
+        await openMenuSafe(driver);
+
+        await driver.clickElement({ text: 'Settings', tag: 'div' });
+        await driver.clickElement({ text: 'Contacts', tag: 'div' });
+
+        await driver.clickElement('.address-book__link');
+
+        const inputUsername = await driver.findElement('#nickname');
+        await inputUsername.fill('#nickname', 'Test User');
+
+        const inputAddress = await driver.findElement(
+          'input[placeholder="Enter public address (0x) or domain name"]',
+        );
+        await inputAddress.fill(
+          '[data-testid="ens-input"]',
+          '0x56A355d3427bC2B1E22c78197AF091230919Cc2A',
+        );
```

Comment:
There would have been a typo here and below does the same action.

```suggestion
        await driver.fill('[data-testid="ens-input"]','0x56A355d3427bC2B1E22c78197AF091230919Cc2A');
```